code/modules/mob/say.dm

```diff
@@ -40,9 +40,15 @@
 		to_chat(usr, "<span class='danger'>Speech is currently admin-disabled.</span>")
 		return
 
-	if(jobban_isbanned(src, "OOC"))
+	var/jb = jobban_isbanned(src, "OOC")
+	if(QDELETED(src))

```

Comment:
This is formatted differently than all the rest and that upsets me.

Reply:
twas a very menial task